hapipheno-client/pom.xml

```diff
@@ -0,0 +1,166 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- one-liner to take you to the cloud with settings form the application.yaml file: -->
+    <!-- 'mvn clean package com.google.cloud.tools:jib-maven-plugin:dockerBuild -Dimage=distroless-hapi && docker run -p 8080:8080 -e spring.batch.job.enabled=false distroless-hapi' -->
+    <!--
+    Note: HAPI projects use the "hapi-fhir" POM as their base to provide easy management.
+    You do not need to use this in your own projects, so the "parent" tag and it's
+    contents below may be removed
+    if you are using this file as a basis for your own project.
+    -->
+    <parent>
+        <groupId>org.monarchinitiative.hapipheno</groupId>
+        <artifactId>hapi-pheno</artifactId>
+        <version>0.4.1</version>
+    </parent>
+
+
+    <artifactId>hapipheno-client</artifactId>
+    <version>0.4.1</version>
+    <packaging>jar</packaging>
+    <properties>
+        <java.version>11</java.version>
+        <spring_boot_version>2.3.4.RELEASE</spring_boot_version>
+        <hapi.version>5.6.2</hapi.version>
+        <protobuf.version>3.16.1</protobuf.version>
+
+    </properties>
+
+
+
+
+    <dependencies>
+        <!-- This dependency includes the core HAPI-FHIR classes -->
+        <dependency>
+            <groupId>ca.uhn.hapi.fhir</groupId>
+            <artifactId>hapi-fhir-client</artifactId>
```

Comment:
comment says "This dependency includes the core HAPI-FHIR classes", artifactId is hapi-fhir-client.
Doesn't make obvious sense to me.